Gemfile

```diff
@@ -27,3 +27,5 @@ gem "rake"
 
 # for unit testing optional sorbet support
 gem "sorbet-runtime"
+
+gem "timecop", "~> 0.9.4"
```

Comment:
Could we use the [ActiveSupport time helpers](https://api.rubyonrails.org/v5.2.5/classes/ActiveSupport/Testing/TimeHelpers.html) instead? The gem already depends on parts of Rails, so that feels like a smaller step than a completely separate gem.